lib/xlsxtream/io/rubyzip.rb

```diff
@@ -1,68 +0,0 @@
-require "zip"
```

Comment:
Please do not remove the RubyZip IO wrapper yet, I'd like to compare compatibility and performance of both wrappers before removing rubyzip.